src/adv.py

```diff
@@ -38,14 +39,83 @@
 #
 
 # Make a new player object that is currently in the 'outside' room.
+player = Player(room['outside'])
+
+"""
+returns the new room that the player moves to if the move was succesful
+or returns to the current room if the move was not successful
+"""
+def try_direction(direction, current_room):
+    attribute = direction + '_to'
+
+    #see if the inputted direction is one we can move to
+    #checks if current room as the attribute that is appended
+    if hasattr(current_room, attribute):
+        #if it has the attribute it gives you the room you are moving to, fetch the new room
+        return getattr(current_room, attribute)
+
+    else:
+        print("You can't go that way!")
+        return current_room
 
 # Write a loop that:
+while True:
 #
 # * Prints the current room name
+    print(player.current_room.name)
+
 # * Prints the current description (the textwrap module might be useful here).
+    print(player.current_room.description)
+
 # * Waits for user input and decides what to do.
+    #handles multiple words
+    s = input("\n> ").lower().split()
```

Comment:
Hahahha, the lower and the split are genius. The next week we need to do is figure out how to parse those split strings.